Assets/MRTK/Core/Inspectors/Setup/MixedRealityProjectConfiguratorWindow.cs

```diff
@@ -368,6 +372,7 @@ private void RenderSelectXRSDKPlugin()
             {
 
 #if UNITY_2020_2_OR_NEWER
+                Debug.Log(MRTKConfiguratorLogPrefix + " Installing the Unity OpenXR Plugin. Operation performed is equivalent to installing the OpenXR plugin manually via Window -> Package Manager -> Packages: Unity Registry.");
```

Comment:
For this one (and any others that are triggered by a button press), I wonder if we want to either

1. Make sure the button's label lets the user know it's about to install something when pressed
2. Pop-up a secondary dialog when the button is present to confirm the user definitely wants to install the package

Reply:
I'd say let's go with option 1.

Reply:
Just made the update. Also I want to mention that I don't think the installation of packages performed by the configurator has a high risk of user complaining as the packages to be installed are all Unity packages.